src/redux/store.js

```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+import bookReducer from './books/booksSlice';
+
+const store = configureStore({
+  reducer: {
+    books: bookReducer,
```

Comment:
- [ ] Kindly consider adding a categories reducer in the store

Reply:
> * [ ]  Kindly consider adding a categories reducer in the store

Thanks for your feedback @omarsalem7 . Changes have been made